app-antivirus/fangfrisch/fangfrisch-1.9.0.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1 readme.gentoo-r1 systemd
+
+DESCRIPTION="Update and verify unofficial Clam Anti-Virus signatures"
+HOMEPAGE="https://github.com/rseichter/fangfrisch https://pypi.org/project/fangfrisch/"
+SRC_URI="https://github.com/rseichter/fangfrisch/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+MY_CONF="/etc/${PN}.conf"
+MY_DBDIR="/var/lib/${PN}"
+DISABLE_AUTOFORMATTING=1
+DOC_CONTENTS="See https://rseichter.github.io/fangfrisch/ for the official
+documentation.
+
+### Fresh installations:
+
+Modify ${MY_CONF} according to your preferences.
+Assuming you place the database into ${MY_DBDIR}
+(recommended), execute the following commands in a root shell:
+
+mkdir -m 0770 ${MY_DBDIR}
+chgrp clamav ${MY_DBDIR}
+sudo -u clamav -- fangfrisch -c ${MY_CONF} initdb
+
+You can now enable /etc/cron.d/${PN} for periodic updates.
+
+### Updating from a previous release:
+
+Either create a fresh database or manually delete all existing
+database tables, then run the initdb command as shown above."
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-python/requests-2.22.0[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.3.11[${PYTHON_USEDEP}]"
```

Comment:
With git log -pC, I get this interdiff between versions.
```diff
-
-DEPEND="
-       >=dev-python/requests-2.22.0[${PYTHON_USEDEP}]
-       >=dev-python/sqlalchemy-1.4.0[${PYTHON_USEDEP}]
-"
+DEPEND=">=dev-python/requests-2.22.0[${PYTHON_USEDEP}]
+       >=dev-python/sqlalchemy-1.3.11[${PYTHON_USEDEP}]"
```

Questions:
- why is this changing style to put the dependencies on the same line as the double quotes? Main difference is that it's kind of a pain if you ever want to add more since each dependency has to be on its own line and therefore you have to change two lines to move the quote character around as well.
- The change itself is to drop the sqlalchemy dependency from >=1.4.0 to >=1.3.11, which I am very suspicious to see in a version *update*. But also it looks like the dependency in https://github.com/rseichter/fangfrisch/blob/master/setup.cfg#L51 has been 1.4.0 for a while, including in the 1.9.0 release, so if it's correct in Gentoo then it is wrong upstream.

Reply:
That was indeed a mistake, SQLAlchemy >= 1.4.0 is required. Fixed.